Include/cpython/longintrepr.h

```diff
@@ -76,8 +76,8 @@ typedef long stwodigits; /* signed variant of twodigits */
     - 1: Zero
     - 2: Negative
 
-   The third lowest bit of lv_tag is reserved for an immortality flag, but is
-   not currently used.
+   The third lowest bit of lv_tag is reserved for an immortality flag, and is
```

Comment:
```suggestion
   The third lowest bit of lv_tag is
```
It's not reserved any more